packages/mask/shared/helpers/checkAndRequestIdentityPermission.ts

```diff
@@ -0,0 +1,10 @@
+import type { Manifest } from 'webextension-polyfill'
+
+export async function checkAndRequestPermission() {
+    const contained = await browser.permissions.contains({ permissions: ['identity'] })
```

Comment:
this helper should check for the environment like `requestPermissionFromExtensionPage` did:

```ts
    if (!isEnvironment(Environment.ExtensionProtocol) || isEnvironment(Environment.ManifestBackground)) {
        // The User Activation limitation is from Firefox
        throw new Error(
            'browser.permissions.request can only be called after a User Activation and from a chrome-extension:// protocol.',
        )
    }
```